client/blocks/comments/post-comment-list.scss

```diff
@@ -13,6 +13,20 @@
 		float: right;
 		margin-top: 4px;
 	}
+
+	.button.comments__manage-comments-button {
+		color: inherit;
+		float: right;
+		margin-top: -2px;
+		margin-right: 14px;
+		padding: 0;
+
+		@include breakpoint-deprecated( '<660px' ) {
```

Comment:
I know that this should be avoided, but I feel like it's necessary in this case to match the styles of the "Follow Conversation" button. The closest other breakpoint appears to be 600px, but changing the Reader to break at 600px instead of 660px is likely a larger issue that's out of scope for this PR.

https://github.com/WordPress/gutenberg/blob/8ac282605512cb4ee0c90973b60e1d01596df018/packages/base-styles/_breakpoints.scss#L11

Reply:
👍 I agree, ideally we'd port all of these in the section together.